bin/test_validator

```diff
@@ -60,24 +62,36 @@ if [[ $i -eq $WAITING ]]; then
     false
 fi
 
-echo Writing pubber output to $PUBBER_OUT
-cmd="bin/pubber $site_path $project_id $device_id $serial_no extraField=prlagle"
-echo $cmd
-$cmd > $PUBBER_OUT 2>&1 &
-
-for i in `seq 1 $WAITING`; do
-    if fgrep "Connection complete" $PUBBER_OUT; then
-        break
+function pubber_bg {
+    did=$1
+    shift
+    outfile=$PUBBER_OUT.$did
+    serial_no=validator-$RANDOM
+    echo Writing pubber output to $outfile, serial no $serial_no
+    cmd="bin/pubber $site_path $project_id $did $serial_no $@"
+    echo $cmd
+    $cmd > $outfile 2>&1 &
+
+    # Give a little bit of time to settle before deterministic check
+    sleep 10
+
```

Comment:
Should we check if the process is still alive after the sleep?

Reply:
The lines immediately following make sure that it started up ok (reaches quiescent state), and any check for did-it-die would be pretty arbitrary after 10 seconds... It's not a failure mode I've seen (successfully complete startup but have died within 10 seconds).  Sometimes I put a check in at the end of the script to say "hey, this died unexpectedly so probably invalid results"... but that's slightly different.